resources/lang/de.json

```diff
@@ -336,6 +336,9 @@
     "settings.request.accept-success": "Folgen-Anfrage erfolgreich akzeptiert.",
     "settings.request.reject-success": "Folgen-Anfrage erfolgreich abgelehnt.",
     "settings.language.set": "Sprache ändern",
+    "settings.visibility": "Sichtbarkeit",
```

Comment:
These translations should also be added in at least english

Reply:
Can't this be done in Weblate by someone wo CAN english? :D